### PR TITLE
point tidyFIA at ncxfiadb database

### PR DESCRIPTION
**Pull request by hrodmn**
We have updated the FIADB clone on a new PostgreSQL database in AWS, this PR switches from the older database to the new one with more recent FIA data and a more true representation of the schema!

fiadb!UNITO-UNDERSCORE!update

### CODE REVIEW

**Review by BKDaugherty — APPROVED** [review]
data-migration-how-hard-can-it-be

File: R/tidy_FIA.R [hunk 1]
```diff
@@ -67,7 +67,7 @@ tidy_fia <- function(states = NULL, aoi = NULL, postgis = TRUE,
     con <- DBI::dbConnect(
       RPostgres::Postgres(),
       dbname = "fiadb",
-      host = "fiadb.csrjp3emmira.us-east-1.rds.amazonaws.com",
+      host = ncxfiadb_host,
       port = 5432,
       user = "tidyfia",
```
Comment (bransonf) [inline R/tidy_FIA.R:72]: Should we use the `readonly` user here?

**Review by bransonf — APPROVED** [review]
Only question is should we use the `readonly` user in this package?

Other than that, looks great

File: R/tidy_FIA.R (hunk 1 above)
Comment (hrodmn) [reply R/tidy_FIA.R:72]: Branson made a 'readonly' ROLE, then granted those permissions to the 'tidyfia' user so I think we are good to go!

Comment (BKDaugherty) [reply R/tidy_FIA.R:72]: 6laow5

Comment (hrodmn) [reply R/tidy_FIA.R:72]: @BKDaugherty made a 'readonly' ROLE, then granted those permissions to the 'tidyfia' user so I think we are good to go!

**Pull request merged**